### PR TITLE
Simplify bearing() method

### PR DESCRIPTION
**Pull request by tmpsantos**
Backport of http://github.com/mapbox/cheap-ruler/pull/29

Patch by: Tobias Bieniek @Turbo87

### CODE REVIEW

File: include/mapbox/cheap_ruler.hpp [hunk 1]
```diff
@@ -103,7 +103,7 @@ class CheapRuler {
             return 0.;
         }
 
-        auto value = std::atan2(-dy, dx) * 180. / M_PI + 90.;
+        auto value = std::atan2(dy, dx) * 180. / M_PI;
```
Comment (Turbo87) [inline include/mapbox/cheap_ruler.hpp]: @tmpsantos you'll need to flip the two parameters in the `atan2()` call

**Review by Turbo87 — CHANGES_REQUESTED** [review]

File: include/mapbox/cheap_ruler.hpp (hunk 1 above)
Comment (tmpsantos) [reply include/mapbox/cheap_ruler.hpp]: ops

**Review by Turbo87 — APPROVED** [review]

**Comment by tmpsantos** [comment]
@Turbo87 thanks for you contribution!

**Review by brunoabinader — APPROVED** [review]

**Comment by brunoabinader** [comment]
Sorry, I thought @Turbo87's :+1: was enough :)

**Pull request merged**